src/BenchmarkDotNet/Configs/DebugConfig.cs

```diff
@@ -82,5 +83,7 @@ public string ArtifactsPath
         public IEnumerable<BenchmarkLogicalGroupRule> GetLogicalGroupRules() => Array.Empty<BenchmarkLogicalGroupRule>();
 
         public ConfigOptions Options => ConfigOptions.KeepBenchmarkFiles | ConfigOptions.DisableOptimizationsValidator;
+
+        public IReadOnlyList<Conclusion> ConfigAnalyse => emptyConclusion;
```

Comment:
`ConfigAnalyse` sounds a little bit confusing. Could we rename it to something like `ConfigConclusions` or `ConfigAnalysisConclusion`?